paychmgr/paych.go

```diff
@@ -54,84 +60,24 @@ func NewManager(sm *stmgr.StateManager, pchstore *Store, api ManagerApi) *Manage
 	}
 }
 
-func maxLaneFromState(st *paych.State) (uint64, error) {
```

Comment:
moved this into state.go and fixed a bug in how it was calculating the max lane (it used to return the max value for the lane if there were no lanes - now it returns zero)